packages/icu4c.rb

```diff
@@ -24,16 +24,15 @@ def self.build
     Dir.chdir 'source' do
       system "mold -run ./configure \
         #{CREW_OPTIONS} \
-        --enable-static \
         --enable-shared \
         --disable-samples \
         --disable-tests"
       system 'make'
     end
   end
 
-  @icuver = '74.2'
-  @oldicuver = %w[73]
+  @icuver = '75.1'
```

Comment:
Use `version` here instead?  Not sure how to handle incrementing old version.  Maybe `@oldicuver = version.split('.').first.to_i - 1`?